src/components/layouts/MainPage.js

```diff
@@ -75,14 +72,14 @@ class MainPage extends React.Component {
                     this.state.introText = confData.introText;
                     this.state.dictionariesPaths = confData.dictionaries;
                     
-                    let bundleAnalyzeReportOnlyAtWebDev = packageJson.homepage.indexOf("web_dev") === -1 ? "" : <Row><a href="report.html" target="_blank">{"See the bundle.js analyze"}</a></Row>;
+                    let bundleAnalyzeReportOnlyAtWebDev = packageJson.homepage.indexOf("web_dev") === -1 ? "" : <Row justify="center"><a href="report.html" target="_blank">{"See the bundle.js analyze"}</a></Row>;
                     
                     this.state.footer = 
                     <Footer key={"footer"} style={{ textAlign: 'center', background: '#fff' }}>
-                        <Row>{confData.footerText1} Find code at
+                        <Row justify="center">{confData.footerText1} Find code at
```

Comment:
maybe modify confData to setup year to 2021 ?

Reply:
![image](https://user-images.githubusercontent.com/25181517/104156412-47155d80-53e9-11eb-8b76-a01c03d43165.png)